pulp_ansible/app/tasks/utils.py

```diff
@@ -12,7 +12,7 @@
 
 def get_api_version(url):
     """Get API version."""
-    result = re.findall(r"/api/v(\d)", url)
+    result = re.findall(r"v(\d+)", url)
```

Comment:
Actually I should not have recommended the `+`, sorry for this. Also it needs slashes on the front and back that @newswangerd suggested on irc, but we didn't bring to you here.

```suggestion
    result = re.findall(r"/v(\d)/", url)
```